dojo_model.py

```diff
@@ -0,0 +1,117 @@
+import random
+from room import Room, Office, Living
+from people import Person, Staff, Fellow
+
+spacer = " "
+
+
+class Dojo (object):
```

Comment:
**Quick note ->** In python 3, '**_object_**' is -_by default_- inherited implicitly. You don't have to import '**_object_**'. every time you create a class. you can simply do **_'class Dojo:'_**